### PR TITLE
hotspot: added custom dns docs entries

### PR DESCRIPTION
**Pull request by Tbaile**
Please review thoroughly, I have no clue on how the tone of the documentation is meant to be.

Ref:
- https://github.com/NethServer/nethsecurity/issues/635

### CODE REVIEW

File: hotspot.rst [hunk 1]
```diff
@@ -93,3 +93,37 @@ Unregister your unit
 If you made some error registering your unit (es. unit was associated to a wrong hotspot instance) or you want to remove this service, do the login in the hotspot section of NethSecurity and click to :guilabel:`Unregister unit`.
 Your unit will be removed both from the NethSecurity and from the remote hotspot manager, the interface used in your Nethsecurity will be freed up and you can use it for other purposes.
 
+Change DNS settings
+^^^^^^^^^^^^^^^^^^^
+
+By default the DNS server used by the hotspot is from OpenDNS, to change the DNS settings manual configuration is required.
+Please follow the steps below from the terminal:
+
+1. Edit the `uci` configuration file with the following commands:
```
Comment (gsanchietti) [inline hotspot.rst:102]:
```suggestion
1. Edit the UCI configuration file with the following commands:
```

File: hotspot.rst [hunk 2]
```diff
@@ -93,3 +93,37 @@ Unregister your unit
...
+   uci set dedalo.config.dns2='<insert dns 2>'
+
+2. Save the changes with the following command:
+
+.. code-block:: bash
+
+   uci commit dedalo
+   reload_config
+
+3. Restart the `dedalo` service with:
```
Comment (gsanchietti) [inline hotspot.rst:116]:
```suggestion
3. Restart the dedalo service with:
```

File: hotspot.rst [hunk 3]
```diff
@@ -93,3 +93,37 @@ Unregister your unit
...
+
+   uci set dedalo.config.dns1='<insert dns 1>'
+   uci set dedalo.config.dns2='<insert dns 2>'
+
+2. Save the changes with the following command:
+
+.. code-block:: bash
+
+   uci commit dedalo
+   reload_config
```
Comment (gsanchietti) [inline hotspot.rst:114]: This should not be needed, it's automatically execute by the restart:
```suggestion
```

**Review by gsanchietti — APPROVED** [review]
Merge when releasing the linked issue

**Review by cotosso — APPROVED** [review]

**Pull request merged**